### PR TITLE
ID-1-HomePage

### PR DESCRIPTION
**Pull request by AshBll**

### CODE REVIEW

**Review by AurelieFqt — DISMISSED** [review]
ok

File: src/Controller/HomeController.php [hunk 1]
```diff
@@ -13,6 +13,6 @@ class HomeController extends AbstractController
      */
     public function index(): Response
     {
-        return $this->render('home/index.html.twig');
+        return $this->render('home/index.html.twig', ['website' => 'AMON']);
```
Comment (RomainClair) [inline src/Controller/HomeController.php]: If it's a constant value, there is no need for a twig variable, just write it in the template

File: templates/home/index.html.twig [hunk 2]
```diff
@@ -1,8 +1,56 @@
...
+{% endblock %}
+
+{% block title %}{{ website }} - Home{% endblock %}
 
 {% block body %}
-    <h1>Welcome on your project 3</h1>
-    <img src="https://media.giphy.com/media/hS58Kb2OOCMPS/giphy.gif" alt="Dewey dancing">
+    <section id='home'>
+        <div class='container_home-menu' id="menu">
+            <img id='home-logo' src="{{ asset('build/images/logo.svg')}}" alt="logo">
```
Comment (RomainClair) [inline templates/home/index.html.twig:13]: Your static images should probably not be in the build directory. May be in public/images ?

File: templates/home/index.html.twig [hunk 3]
```diff
@@ -1,8 +1,56 @@
...
+    {{ encore_entry_link_tags('home') }}
+{% endblock %}
+
+{% block title %}{{ website }} - Home{% endblock %}
 
 {% block body %}
-    <h1>Welcome on your project 3</h1>
-    <img src="https://media.giphy.com/media/hS58Kb2OOCMPS/giphy.gif" alt="Dewey dancing">
+    <section id='home'>
+        <div class='container_home-menu' id="menu">
```
Comment (RomainClair) [inline templates/home/index.html.twig:12]: A menu should be a nav

**Review by RomainClair — CHANGES_REQUESTED** [review]
In general, think about using semantic HTML tags and keep the div only when no semantic tag is possible. Use section, nav, article, etc.

**Review by AurelieFqt — DISMISSED** [review]

**Review by KevinRoig — DISMISSED** [review]

File: src/Controller/HomeController.php (hunk 1 above)
Comment (KevinRoig) [reply src/Controller/HomeController.php]: Ok done

File: templates/home/index.html.twig (hunk 2 above)
Comment (KevinRoig) [reply templates/home/index.html.twig:13]: OK done

File: templates/home/index.html.twig (hunk 3 above)
Comment (KevinRoig) [reply templates/home/index.html.twig:12]: OK done

**Review by KevinRoig — APPROVED** [review]
ok

**Review by RomainClair — APPROVED** [review]
It is strange to have a div id menu that contains a nav instead of a nav id menu. Furthermore there's is a link outside the nav but in the menu.
And still, static images (like the logo and the favicon that are not build should probably not be in the build directory

**Pull request merged**